lua/telescope/builtin.lua

```diff
@@ -501,32 +501,21 @@ builtin.builtin = function(opts)
 
   local objs = {}
 
-  for k, v in pairs(builtin) do
-    local debug_info = debug.getinfo(v)
-
-    table.insert(objs, {
-      filename = string.sub(debug_info.source, 2),
-      lnum = debug_info.linedefined,
-      col = 0,
-      text = k,
-
-      start = debug_info.linedefined,
-      finish = debug_info.lastlinedefined,
-    })
+  for k, _ in pairs(builtin) do
```

Comment:
Can you actually leave the debut stuff? It makes it so we can show a preview of the code, which I think is pretty cool!

Reply:
Done. I also threw in an extra change to keep insert mode when selecting a builtin, if you don't mind